features/step_definitions/api_request.rb

```diff
@@ -123,6 +123,18 @@
   @outgoings_data = { "outgoings": ["name": string, "payments": table.hashes.map { cast_values(_1) }] }
 end
 
+Given("I add outgoing details for {string} of {int} per month") do |string, monthly_amount|
```

Comment:
@starswan What you you think of creating payment dates based on the submission date?
Eg:
https://github.com/ministryofjustice/cfe-civil/blob/7a8ee3f3487d5ec1853ae66747f189cc4d43657b/features/step_definitions/api_request.rb#L179

Reply:
not sure its worth it unless we need to